src/libtrx/game/savegame/common.c

```diff
@@ -425,8 +415,15 @@ void Savegame_PersistGameToCurrentInfo(const GF_LEVEL *const level)
     } else {
         resume->equipped_gun_type = lara->gun_type;
     }
-    resume->gun_status = LGS_ARMLESS;
 #endif
+
+    resume->holsters_gun_type = lara->holsters_gun_type;
+    resume->back_gun_type = lara->back_gun_type;
+    if (TR_VERSION == 1 && lara->gun_status == LGS_READY) { // TODO: TR2
```

Comment:
Can you please elaborate on this particular condition? I'm not fully following

Reply:
In TR1, if Lara ends a level with guns out, she starts the next level with guns out. In TR2 she always starts with guns away. The to do is to come back and see about getting it to work in TR2 and perhaps offering an option (I think it may interfere with offshore rig start, will need to investigate).